src/app/projects/components/project-settings/helper/label-helper.ts

```diff
@@ -75,8 +75,11 @@ export class LabelHelper {
 
         labelInput.value = '';
         labelInput.focus();
-        this.settings.requestTimeOut = true;
-        timer(100).subscribe(() => this.settings.requestTimeOut = false);
+        requestTimeOut = true;
+        timer(100).subscribe(() => {
+            requestTimeOut = false;
+            return requestTimeOut;
```

Comment:
this is not a return for the add label function but for the provided anonymous function in {}